chaoslib/litmus/pod_memory_hog/app_memory_stress.j2

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: app-memory-stress-{{ run_id }}
+  labels:
+    app: app-memory-stress
+    name: app-memory-stress-{{ run_id }}  
+{% if chaos_uid is defined and chaos_uid != '' %}        
+    chaosUID: {{ chaos_uid }}
+{% endif %}   
+spec:
+  nodeName: {{ app_node }}
+  restartPolicy: Never
+  containers:
+  - image: {{ memory_stress_image }} 
+    imagePullPolicy: Always
+    name: app-memory-stress
+    command: 
+    - /bin/sh
+    args:
+    - ./memory-hog.sh 
+    env:
+        ## mandatory: provide id of docker container
+      - name: CONTAINER_ID
+        value: "{{ c_container_id }}"
+        ## optional: provide duration of memory chaos 
+        ## default: 60 (in sec)
+      - name: DURATION
+        value: "{{ c_duration }}"
+        ## optional: provide amount of memory to be consumed in container
+        ## default: 500M (in Megabytes)
+      - name: MEMORY_CONSUMPTION
+        value: "{{ m_consumption }}"
+{% if ramp_time is defined and ramp_time != '' %}
+      - name: RAMP_TIME
```

Comment:
Why we are passing ramp time here? We already have ramp time in before& after chaos injection section?

Reply:
There we are waiting for a certain duration after the job is succeeded. Here the ramp time is used to wait for the memory to come back in a normal state after hogging. @ispeakc0de 
Removed from those places.